zephyr/esp32/include/sdkconfig.h

```diff
@@ -1,3 +1,16 @@
+#ifdef CONFIG_MCUBOOT
+
+#define BOOTLOADER_BUILD 1
+#define NDEBUG 1
+
+#define CONFIG_BOOTLOADER_FLASH_XMC_SUPPORT 1
+#define CONFIG_BOOTLOADER_WDT_TIME_MS 9000
```

Comment:
I think both this CONFIGs should not be hardcoded. It can be added into kconfig on main branch.